[WIP] Activate doctest (#3503)
qgis/QGIS-Documentation: readme.rst

  * Contributor: Sorin Călinică < sorin.calinica[at]gmail.com >
  * Contributor: Tudor Bărăscu < tudor.barascu[at]qtibia.ro >

Testing Python snippets
***********************

> To test Python snippet of code, you need a QGIS installation, for this there is many options:

Reviewer: ```suggestion
To test Python code snippets , you need a QGIS installation. For this there are many options:
```

You can use your system version of QGIS::

   make -f venv.mk doctest

> You can use a manually builded installation of QGIS, to to so, you need to create a custom Makefile

Reviewer: ```suggestion
You can use a manually built installation of QGIS. To do so, you need to create a custom Makefile
```

> extention on top of ``venv.mk`` file, for example a ``user.mk`` file with following content::

Reviewer: ```suggestion
extension on top of the ``venv.mk`` file, for example a ``user.mk`` file with the following content::
```
